### DISCUSSION TITLE
How to use a tag in text - Conventions

### DISCUSSION THREAD

**Discussion by INFO-studio** [squidfunk/mkdocs-material#7240] [Q&A] [answered]
Just like the example in [https://squidfunk.github.io/mkdocs-material/conventions/](https://squidfunk.github.io/mkdocs-material/conventions/)
I want to create my own tag symbols and use it on my site
I find that in the code of markdown it's like `<!-- md:version 8.3.0 --> `
Apologize for my silly question, But I can't find any tutorials and any yml settings about it.

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @INFO-studio,
it's a custom feature of the material docs, you can implement it yourself. This has been asked many times over in the discussions. 
Search for conventions, tags, symbols etc. Closing as duplicate:
- https://github.com/squidfunk/mkdocs-material/discussions/6544
